.github/workflows/tests.yml

```diff
@@ -39,3 +39,15 @@ jobs:
 
     - name: Execute integration-tests
       run: make integrationtest
+
+    - name: Archive logs
+      uses: actions/upload-artifact@v2
+      with:
+          name: logs
+          # This directory is created by the fixture session_tmppath. The path is composed of a
+          # prefix defined by --basepath, followed by a folder name, in this case it is "karapace".
+          #
+          # See:
+          # - https://docs.pytest.org/en/6.2.x/tmpdir.html#base-temporary-directory
+          # - fixture session_tmppath in tests/integration/conftest.py
+          path: /tmp/pytest*
```

Comment:
I have tried:

- `/tmp`: didn't work because not all files have read permission
- `/tmp/pytest-of-*/karapace`: not sure why it did not work
- `/tmp/pytest*`

Reply:
The files are inside `/tmp/pytest-of-runner/pytest-<number>/popen-<worker>/karapacecurrent`

Reply:
These directories contained the Kafka data files, which are pre allocated files. So the artifact was around 10GB of data. This PR made the directory for log files configurable and separate from the data directory